lib/readline.js

```diff
@@ -843,7 +843,7 @@ Interface.prototype._ttyWrite = function(s, key) {
         if (this.listenerCount('SIGTSTP') > 0) {
           this.emit('SIGTSTP');
         } else {
-          process.once('SIGCONT', (function(self) {
+          process.once('SIGCONT', (function continueProcess(self) {
```

Comment:
So we could actually do a slightly better thing here. Instead of the IIFE, we could use an arrow function and replace `self` with `this` — unless I'm missing something. Then we will just have a normal event callback instead of this weird thing it is right now.  Feel free to either do that change here or in a separate PR.

Reply:
Yes that make sense, may be will update the same in another PR. Thanks for your time on this.